apps/condo/domains/marketplace/components/MarketplaceOnboarding/OfferSetupPage.tsx

```diff
@@ -0,0 +1,141 @@
+import get from 'lodash/get'
+import { useRouter } from 'next/router'
+import React, { useCallback, useEffect } from 'react'
+
+import { useOrganization } from '@open-condo/next/organization'
+import { Typography } from '@open-condo/ui'
+
+import { CONTEXT_FINISHED_STATUS, CONTEXT_IN_PROGRESS_STATUS } from '@condo/domains/acquiring/constants/context'
+import { AcquiringIntegrationContext as AcquiringContext, AcquiringIntegration } from '@condo/domains/acquiring/utils/clientSchema'
+import { Loader } from '@condo/domains/common/components/Loader'
+import { extractOrigin } from '@condo/domains/common/utils/url.utils'
+import { INVOICE_CONTEXT_STATUS_INPROGRESS } from '@condo/domains/marketplace/constants'
+import { InvoiceContext } from '@condo/domains/marketplace/utils/clientSchema'
+import { IFrame } from '@condo/domains/miniapp/components/IFrame'
+import { MANAGING_COMPANY_TYPE } from '@condo/domains/organization/constants/common'
+
+type SetupAcquiringProps = {
+    onFinish: () => void
+}
+
+export const OfferSetupPage: React.FC<SetupAcquiringProps> = ({ onFinish }) => {
+    const router = useRouter()
+    const { organization } = useOrganization()
+    const orgId = get(organization, 'id', null)
+    const orgType = get(organization, 'type', MANAGING_COMPANY_TYPE)
+
+    const createAction = AcquiringContext.useCreate({
+        status: CONTEXT_IN_PROGRESS_STATUS,
+        settings: { dv: 1 },
+        state: { dv: 1 },
+    })
+    const updateAction = AcquiringContext.useUpdate({
```

Comment:
there is no need to update the status, this is done in afterChange of the Accept scheme